### PR TITLE
Add Ada Modeling framework, SOAP and Servlet server

### PR DESCRIPTION
**Pull request by reznikmm**

### CODE REVIEW

File: index/ma/matreshka_amf.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+authors = ["Vadim Godunko"]
+website = "https://forge.ada-ru.org/matreshka"
+maintainers = ["Vadim Godunko <vgodunko@gmail.com>", "Maxim Reznik <reznikmm@gmail.com>"]
+maintainers-logins = ["godunko", "reznikmm"]
+
+project-files = ["build_matreshka_amf.gpr"]
+
+    [general.depends-on]
+    matreshka_league = "any"
+    matreshka_league_xml = "any"
```
Comment (mosteo) [inline index/ma/matreshka_amf.toml]: @reznikmm This dependency doesn't seem to exist? Would it be matreshka_xml?

**Review by mosteo — CHANGES_REQUESTED** [review]

File: index/ma/matreshka_amf.toml (hunk 1 above)
Comment (reznikmm) [reply index/ma/matreshka_amf.toml]: @mosteo Indeed. Should be fixed now.

File: index/ma/matreshka_amf_uml.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+maintainers-logins = ["godunko", "reznikmm"]
+
+project-files = ["build_matreshka_amf_uml.gpr"]
+
+    [general.depends-on]
+    matreshka_league = "any"
+    matreshka_amf_dd = "any"
+
+['18.1']
+origin = "https://github.com/reznikmm/matreshka-alire/archive/matreshka_amf_uml-18.1.tar.gz"
```
Comment (mosteo) [inline index/ma/matreshka_amf_uml.toml:18]: Hi @reznikmm, I'm getting a 404 on this file. I took a look in the repo and it seems indeed to be missing.

File: index/ma/matreshka_amf.toml (hunk 1 above)
Comment (mosteo) [reply index/ma/matreshka_amf.toml]: It works, thanks.

File: index/ma/matreshka_amf_uml.toml (hunk 2 above)
Comment (reznikmm) [reply index/ma/matreshka_amf_uml.toml:18]: @mosteo Indeed, it should be fixed now.
Can you share, how can I check these packages before push PR?

Comment (mosteo) [reply index/ma/matreshka_amf_uml.toml:18]: What I normally do is:

1. Clone the alire-index repo locally
1. Add it with `alr index --add <path> --name local`
   1. Optionally remove the community index with `alr index --del`
   1. You can have alternative configs with `-c` or setting `ALR_CONFIG` env var
1. Edit the new crate/release within the cloned index
1. Test with `alr list`, `alr show`, `alr get --build`
1. Push changes when happy.

For your submissions, I merely check out the branch to be merged.

Eventually this should be much automated with `publish`, and also vetted during PR automatic checks.

Thanks for the fixes, I should get around to merging it soon.

**Comment by mosteo** [comment]
Merged, thanks!

**Pull request merged**